docs/user-guide/cluster-docker-consul.md

```diff
@@ -0,0 +1,278 @@
+# Clustering / High Availability on Docker Swarm with Consul
+
+This guide explains how to use Træfik in high availability mode in a Docker Swarm and with Let's Encrypt.
+
+Why we need Traefik in cluster mode? Running multiple instances should work out of the box?
```

Comment:
Why **do** we need

> Running multiple instances should work out of the box?
Not sure to understand your sentence. Do you mean _How a cluster shoud work out of the box?_ ?

Reply:
> Running multiple instances should work out of the box?
It's because you can run multiple instances of Traefik and it works but you miss some stuff like sharing challenge for LE and more. And it's what I want to explain in this guide: you can't just start traefik instances and hope it will magically work with LE.